src/components/global/Playground/index.tsx

```diff
@@ -105,20 +133,27 @@ export default function Playground({
     console.warn('No code usage examples provided for this Playground example.');
     return;
   }
+  if (typeof mode !== 'undefined' && mode !== 'ios' && mode !== 'md') {
+    console.warn(`Invalid mode provided: ${mode}. Accepted values are: "ios" or "md".`);
+    return;
```

Comment:
Do we want to return here? Seems like we could fall back to default behavior to fail more gracefully instead.

Reply:
I think returning makes the most sense so that we do not commit bad playground examples that use an invalid mode. 

The intention of using `mode` would be to restrict the example to one or the other, so falling back would likely result in showing an example that is confusing to users.

With returning early, the playground would not render (with an error logged to the console). This would hopefully be caught in local development and for sure would be caught in a PR.

Reply:
Makes sense to me 👍